core/templates/pages/splash-page/splash-page.component.ts

```diff
@@ -51,7 +51,7 @@ angular.module('oppia').component('splashPage', {
       ctrl.onClickBrowseLibraryButton = function() {
         SiteAnalyticsService.registerClickBrowseLibraryButtonEvent();
         $timeout(function() {
-          WindowRef.nativeWindow.location = '/community-library';
+          WindowRef.nativeWindow.location = '/learn/math';
```

Comment:
Just a thought: The home page with the button "Browse our lessons" redirects to a page which is all about Maths, sound like the Oppia platform only has maths lessons. (I think "Browse our lessons" should redirect to a page where there's a wide variety of subjects for individuals to pick. If this button redirects to the maths classroom then it should say "Browse our Maths lessons" maybe?)

Reply:
To be honest, I'm fine with keeping it as "Browse our lessons" since these lessons were created by us and they're the ones we're specificallly looking to promote. If people feel strongly about making it Browse our Math Lessons, that seems fine to me as well but overall I don't think it's necessary to change.

Reply:
Yep, makes sense. Keeping as is for now. Thanks!